docs/docsite/rst/network/dev_guide/developing_resource_modules_network.rst

```diff
@@ -558,7 +558,7 @@ The following example walks through the integration tests for the ``vyos.vyos.vy
 
 ``test/integration/targets/vyos_l3_interfaces/tests/cli/overridden.yaml``
 
-.. code-block:: yaml
+.. code-block:: yaml+jinja
 
   ---
   - debug:
```

Comment:
The `msg` in the line below seems to be broken (indentation needs fixing, and the `}}` should go into the previous line).

Reply:
Thanks for catching this @felixfontein I put that into a `.yml` file and ran yamllint against it. Hopefully what's there is good now but I'd appreciate if you could take another pass at it.

This is really why I advocate for the `include` directive and externalizing the contents of code blocks to the native file type. YAML embedded in RST is kind of nasty.

Reply:
It's not *that* hard to write a checker which uses docutils to load a RST file and checks all YAML snippets in it with yamllint (similar to what ansible-test is doing in its yamllint test, where it puts the YAML documentation included in a Python module through yamllint). It's just that someone has to implement it :)

Maybe I'll take a look at that once, if nobody else beats me to it...

Reply:
that would be really useful if you could implement it @felixfontein

Reply:
See https://github.com/ansible/ansible-documentation/pull/2385